tests/periph_i2c/tests/04__periph_i2c_flags.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation       Tests to verify correct error codes are given.
+
+Suite Setup         Run Keywords    PHILIP Reset
+...                                 RIOT Reset
+...                                 API Firmware Should Match
+Test Setup          Run Keywords    PHILIP Reset
+...                                 RIOT Reset
+...                                 API Sync Shell
+...                                 I2C Acquire
+Test Teardown       I2C Release
+
+Resource            periph_i2c.keywords.txt
+Resource            api_shell.keywords.txt
+Resource            riot_base.keywords.txt
+
+Force Tags          periph  i2c
+
+*** Test Cases ***
+Read Byte with NOSTOP Flag Should Succeed
+    [Documentation]                         Verify DUT does not lockup if read_byte with NOSTOP
+    API Call Should Succeed                 I2C Read Byte  flag=4
```

Comment:
maybe we can use variables for the flag params instead of *magic* numbers? I mean: add a variable section above the test cases, like:

```
*** Variables ***
${I2C_FLAG_NOSTOP}         4
```

add for `8` and `12`, too.

Reply:
Sure

Reply:
much better now 😄